app/package.json

```diff
@@ -12,6 +12,7 @@
   },
   "dependencies": {
     "@react-native-async-storage/async-storage": "^1.15.7",
+    "@react-native-community/datetimepicker": "^6.1.3",
```

Comment:
It looks like this one isn't being used now?  If so, could you remove it?

Reply:
Done